Mathlib/RingTheory/Ideal/KrullDimension.lean

```diff
@@ -21,7 +21,9 @@ Key results in this file include:
 3. the Krull dimension of any quotient ring of a commutative ring must be less than or equal to
    the Krull dimension of that commutative ring;
 4. two isomorphic commutative rings have the same Krull dimension;
-5. the Krull dimension of a field is zero.
+5. for a commutative ring whose Krull dimension is zero, any prime ideal is maximal;
+6. for a commutative ring whose Krull dimension is zero, any prime ideal is minimal;
+7. the Krull dimension of a field is zero.
```

Comment:
This part of the module docstring should probably follow the conventions here https://leanprover-community.github.io/contribute/doc.html#header-comment , now there are so many key results. You should make a new `## Main statements` subheading and list the statements together with what the theorems are called in Lean. See e.g. `RingTheory/Ideal/IsPrincipal.lean` or many other mathlib files for the formatting you should use, ``* `theorem_name`: theorem_explanation``. Then it will display nicely in the online API docs.